source/retryable-writes/tests/retryableErrors.json

```diff
@@ -9,16 +9,22 @@
       "x": 22
     }
   ],
-  "minServerVersion": "3.7.9",
+  "minServerVersion": "4.1.0",
```

Comment:
Why 4.1.0? Shouldn't it be 4.0.0?

Reply:
Or maybe 4.0.0-rc1?

Reply:
There has not been a 4.0 release with the "failCommands" feature, see the comment in YAML
```
# TODO: this should change to 4.0.0 once a 4.0 version is released that has
# SERVER-35004.
```

Reply:
Sorry. I only looked at the JSON (assuming it was the same as the YAML, which it is except that the comments are lost).

Apparent 4.0.0 latest already supports this, as the tests are failing on Evergreen.

Reply:
We can change it to 4.0.0-rc2 (or just 4.0.0) once rc2 is released.